tests/test_dataset.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
Can you please use the `test_streaming.py` file instead of creating a new one? The `test_streaming.py` file directly tests the `dataset.py` functionality.

Reply:
yep moving this now!